traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/cache/Cache.java

```diff
@@ -18,11 +18,7 @@
 import java.net.Inet6Address;
 import java.net.InetAddress;
 import java.net.UnknownHostException;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
nit: I think intellij probably did this for you, but I think we pretty consistently try to use explicit imports of only what we use

Reply:
Using explicit imports in 9e680217f (and increased Intellij's threshold for using wildcard imports to 9999 classes, because -1 just always uses it)